### ISSUE TITLE
CNN Fine-Tuning using MatConvNet

### ISSUE THREAD

**Issue by kilickaya** [vlfeat/matconvnet#275]
CNN Fine-Tuning aims at tuning CNN parameters (of your model) for your specific problem. For an example fine-tuning using Caffe library can be found from:  

https://github.com/BVLC/caffe/tree/master/examples/finetune_flickr_style

I wonder if a similar mechanism to fine tune already trained models is implemented for MatConvNet? If not, what could be possible steps to do so?

**Comment by omriFdna** [comment]
Hi, 

The steps are very much like in caffe,  you should load the model you want to fine tune and change for example the class layer in the end to your number of classes, take a look at the examples of how to use add_block in order to create a new layer with random weights.  Then start training using cnn_train. 

You can also effect the parameters in the input opts, for example if you want to change learning rate. 

Good luck

**Comment by kilickaya** [comment]
What I understand so far from fine-tuning a CNN using MatConvNet is:

1. Convert your data to MatConvNet format 
2. initialize a CNN model that outputs the same number of classes of your own problem. 
3. train model and see what's going on!
